code/core/src/phone/java/com/adobe/marketing/mobile/services/ui/message/views/MessageFrame.kt

```diff
@@ -58,22 +65,35 @@ internal fun MessageFrame(
     onCreated: (WebView) -> Unit,
     onDisposed: () -> Unit
 ) {
-    val currentConfiguration = LocalConfiguration.current
+    // The current context is the activity that is hosting the message. We can safely cast it to
+    // an Activity because this composable is always used within the context of the activity in
+    // the current implementation of UIService
+    val currentActivity = LocalContext.current as? Activity ?: run {
+        onDisposed()
+        Log.debug(ServiceConstants.LOG_TAG, "MessageFrame", "Unable to get the current activity. Dismissing the message.")
+        return
+    }
```

Comment:
Can you use this approach for finding the activity as it is safer https://github.com/adobe/aepsdk-assurance-android/blob/main/code/assurance/src/main/java/com/adobe/marketing/mobile/assurance/internal/ui/ContextExt.kt#L21